packages/web/components/transactions/transaction-details.tsx

```diff
@@ -54,6 +66,14 @@ export const TransactionDetailsContent = ({
 
   const { logEvent } = useAmplitudeAnalytics();
 
+  const status = transaction.code === 0 ? "success" : "failed";
+
+  const title = {
```

Comment:
Since `transaction.code` determine both the value of `title` and the value of `status` maybe use a useMemo to extract both? instead of `title[status]` where you are also creating the title object each time the component is rendering.